src/views/AboutUs/AboutUs.scss

```diff
@@ -1,47 +1,18 @@
-@import "../../styles/breakpoints";
+@import "../../GlobalSass/mixin.scss";
```

Comment:
Please use the new breakpoints import instead of GlobalSass

Reply:
I changed it and pushed it.